packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx

```diff
@@ -213,32 +89,68 @@ const ErrorFormTooltip = ({
     };
   }, [onShown, isHelp, helpClosable]);
 
+  const isVertical =
+    (attrs.popper && attrs.popper["data-popper-placement"] === "top-start") ||
+    (attrs.popper && attrs.popper["data-popper-placement"] === "top-end")
+      ? "bottom"
+      : "top";
```

Comment:
`attrs.popper?["data-popper-placement"] === "top-end"`

Reply:
@oreqizer suggestion did not work. Was throwing error
```
This comparison appears to be unintentional because the types 'string[]' and 'string' have no overlap.ts(2367)
This condition will always return 'false' since JavaScript compares objects by reference, not value.ts(2839)
```

Ended up doing
```js
const isVertical =
  attrs.popper &&
  (attrs.popper["data-popper-placement"] === "top-start" ||
    attrs.popper["data-popper-placement"] === "top-end")
    ? "bottom"
    : "top";
```

Reply:
🤔 really? that's weird, this didn't work?

```ts
const isVertical =
    attrs.popper?.["data-popper-placement"] === "top-start" ||
    attrs.popper?.["data-popper-placement"] === "top-end"
      ? "bottom"
      : "top";
```

I mean it is just syntax here since we're not enforcing [prefer nullish coalescing](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md) in ESlint, so no problem, I just find it interesting